applications/sink/zeromq-sink/pom.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>zeromq-sink</artifactId>
+    <name>zeromq-sink</name>
+    <description>0MQ sink apps</description>
+    <packaging>jar</packaging>
+
+    <parent>
+        <groupId>org.springframework.cloud.stream.app</groupId>
+        <artifactId>stream-applications-core</artifactId>
+        <version>3.1.0-SNAPSHOT</version>
+        <relativePath/>
+    </parent>
+
+    <dependencies>
+
+        <!-- Spring Cloud Function dependencies -->
+        <dependency>
+            <groupId>org.springframework.cloud.fn</groupId>
+            <artifactId>zeromq-consumer</artifactId>
+        </dependency>
+
+        <!-- Test dependencies -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.junit.vintage</groupId>
+                    <artifactId>junit-vintage-engine</artifactId>
```

Comment:
You know, it looks like we don't need to exclude this dep any more.
It is only like this: `testRuntimeOnly("org.junit.vintage:junit-vintage-engine")`. Therefore is not visible for us at all.